### ISSUE TITLE
DOC: Add details description of the tasks in the README

### ISSUE THREAD

**Issue by celprov** [TheAxonLab/HCPh-fMRI-tasks#53]

**Comment by celprov** [comment]
@esavary did we change some parameters in the visual grating pattern compared to Harvey's implementation? (I'm not 100% sure how to extract the parameters that she reports from psychopy, but if we didn't change anything then her parameters are ours)

**Comment by oesteban** [comment]
I think I had changed some before Elodie started to work on this.

This repo is a fork of that task, so perhaps you can figure out by digging
up the git history.

**Comment by celprov** [comment]
Yep that's exactly what I did. It does not seem we changed anything, except the fixation point is a point and not a cross.
